master/internal/api_experiment.go

```diff
@@ -1301,8 +1301,8 @@ func (a *apiServer) PatchExperiment(
 			go func() {
 				err = runCheckpointGCTask(
 					a.m.system, a.m.rm, a.m.db, taskID, modelExp.JobID, modelExp.StartTime,
-					taskSpec, modelExp.ID, modelExp.Config, checkpoints, []string{fullDeleteGlob}, true,
-					agentUserGroup, user, nil,
+					taskSpec, modelExp.ID, modelExp.Config, checkpoints, []string{fullDeleteGlob}, false,
```

Comment:
I think the first false might need to be true to delete tensorboards? I see it look like it got flipped maybe unintentionally?